docs/install_guides/installing-openvino-vcpkg.md

```diff
@@ -64,6 +64,34 @@ Installing OpenVINO Runtime
 Note that the vcpkg installation means building all packages and dependencies from source, 
 which means the compiler stage will require additional time to complete the process. 
 
+IMPORTANT: In case you are building OpenVINO as dynamic libraries and want to use one of the Paddle, TensorFlow or ONNX frontends, it is required to create [custom vcpkg](https://learn.microsoft.com/en-us/vcpkg/users/triplets#per-port-customization) triplet file `<VCPKG_ROOT>/triplets/community/x64-linux-relelase-dynamic.cmake`, which builds `protobuf` dependency statically:
```

Comment:
```suggestion
IMPORTANT: In case you are building OpenVINO as dynamic libraries and want to use one of the Paddle, TensorFlow or ONNX frontends, it is required to create [custom vcpkg](https://learn.microsoft.com/en-us/vcpkg/users/triplets#per-port-customization) triplet file `<VCPKG_ROOT>/triplets/community/x64-linux-release-dynamic.cmake`, which builds `protobuf` dependency statically:
```